README.md

```diff
@@ -1 +1,25 @@
 # rosetta-be
+## Local Setup Instructions
+
+1. Set up virtual environment:
+```
+$ python3 -m venv venv
+$ . venv/bin/activate
+```
+2. Install dependencies:
+`$ pip install -r requirements.txt`
+
+3. Set environment variables:
+```
+$ export FLASK_APP=flaskr/app
+$ export FLASK_ENV=development
+```
+4. Migrate database:
+```
+$ python flaskr/manage.py db init
+$ python flaskr/manage.py db migrate
```

Comment:
These first two aren't necessary for running the migrations, they're part of the setup process; see my version of the README, we'll need to merge ours together